worklist/views.py

```diff
@@ -0,0 +1,45 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+from django.shortcuts import render
+from worklist.models import WorkList
+
+
+# View to create a worklist
+def create_worklist(request):
+    content = {}
+    if request.method == 'POST':
```

Comment:
Handle non-POST as well please, it probably makes sense to just redirect to the initial page.

Reply:
non-POST is handled, such that in all requests other than POST, content will be empty dictionary, which will be checked in the template (create-worklist.html) and empty form to create a worklist will be rendered!. 
Example, If you hit this url : https://tools.wmflabs.org/worklist-tool/create-worklist , that will be a GET request and a form to create-worklist will be shown!